bin/helpers

```diff
@@ -165,6 +165,14 @@ createPowerFlexPool() (
     powerflex.user.password="${POWERFLEX_PASSWORD}"
 )
 
+# createCertificateAndKey: creates a new key pair.
```

Comment:
That looks like the `gen_cert_and_key()` from https://github.com/canonical/lxd/blob/main/test/includes/certificates.sh which is good but I'd also copy the other function to extract the fingerprint (`cert_fingerprint()`) as it seems like a useful helper. Maybe with the additional option of returning a partial fingerprint?

Reply:
Thanks I'll move it over too. Wasn't aware of it.

Reply:
Fixed in https://github.com/canonical/lxd-ci/pull/128#event-12354900680